### PR TITLE
obis observer. clean commands list

### PR DESCRIPTION
**Pull request by Pnom**

### CODE REVIEW

File: tests/obis-observer/message.test.ts.bak [hunk 1]
```diff
@@ -0,0 +1,130 @@
+/* eslint-disable @typescript-eslint/no-unsafe-member-access */
```
Comment (DarkPark) [inline tests/obis-observer/message.test.ts.bak:1]: redundant file

File: src/obis-observer/commands/downlink/GetObisIdList.ts [hunk 2]
```diff
@@ -42,19 +36,13 @@ const examples: TCommandExampleList = [
...
  * const parameters = {
  *     requestId: 3,
- *     meterProfileId: 5,
- *     index: 0,
- *     obis: {
- *         c: 0,
- *         d: 9,
- *         e: 1
- *     }
+ *     meterProfileId: 5
```
Comment (DarkPark) [inline src/obis-observer/commands/downlink/GetObisIdList.ts:39]: index is missing and hex seems wrong

File: src/obis-observer/commands/downlink/GetObisIdList.ts [hunk 3]
```diff
@@ -77,28 +65,16 @@ class GetObisIdList extends Command {
...
-
-        buffer.setUint8(requestId);
-        buffer.setUint8(meterProfileId);
-        buffer.setUint8(index);
-        buffer.setObis(obis);
-
-        return Command.toBytes(COMMAND_ID, buffer.toUint8Array());
+        return Command.toBytes(
+            COMMAND_ID,
+            new Uint8Array([this.parameters.requestId, this.parameters.meterProfileId, this.parameters.index])
```
Comment (DarkPark) [inline src/obis-observer/commands/downlink/GetObisIdList.ts:76]:
```suggestion
            const {parameters} = this;

            new Uint8Array([parameters.requestId, parameters.meterProfileId, parameters.index])
```
?

File: src/obis-observer/commands/downlink/RemoveObis.ts [hunk 4]
```diff
@@ -31,23 +33,23 @@ const examples: TCommandExampleList = [
  *
  * @example
  * ```js
- * import AddMeterProfile from 'jooby-codec/obis-observer/commands/downlink/AddMeterProfile.js';
+ * import RemoveObis from 'jooby-codec/obis-observer/commands/downlink/RemoveObis.js';
  *
  * const parameters = {
  *     requestId: 3,
  *     meterProfileId: 17
```
Comment (DarkPark) [inline src/obis-observer/commands/downlink/RemoveObis.ts:40]: wrong example and hex

File: src/obis-observer/commands/downlink/SetupObis.ts [hunk 5]
```diff
@@ -0,0 +1,156 @@
...
+
+    static readonly examples = examples;
+
+    static readonly hasParameters = true;
+
+
+    // data - only body (without header)
+    static fromBytes ( data: Uint8Array ) {
+        const buffer = new CommandBinaryBuffer(data);
+
```
Comment (DarkPark) [inline src/obis-observer/commands/downlink/SetupObis.ts:115]:
```suggestion
```

File: src/obis-observer/commands/uplink/GetObisIdListResponse.ts [hunk 6]
```diff
@@ -68,8 +76,8 @@ class GetObisIdListResponse extends Command {
     // data - only body (without header)
     static fromBytes ( data: Uint8Array ) {
         const buffer = new CommandBinaryBuffer(data);
-
         const requestId = buffer.getUint8();
+
```
Comment (DarkPark) [inline src/obis-observer/commands/uplink/GetObisIdListResponse.ts:80]:
```suggestion
```

File: tests/obis-observer/message.test.ts.bak (hunk 1 above)
Comment (Pnom) [reply tests/obis-observer/message.test.ts.bak:1]: fixed

File: src/obis-observer/commands/downlink/GetObisIdList.ts (hunk 2 above)
Comment (Pnom) [reply src/obis-observer/commands/downlink/GetObisIdList.ts]: fixed

File: src/obis-observer/commands/downlink/GetObisIdList.ts (hunk 3 above)
Comment (Pnom) [reply src/obis-observer/commands/downlink/GetObisIdList.ts]: fixed

File: src/obis-observer/commands/downlink/RemoveObis.ts (hunk 4 above)
Comment (Pnom) [reply src/obis-observer/commands/downlink/RemoveObis.ts]: fixed

File: src/obis-observer/commands/downlink/SetupObis.ts (hunk 5 above)
Comment (Pnom) [reply src/obis-observer/commands/downlink/SetupObis.ts]: fixed

File: src/obis-observer/commands/uplink/GetObisIdListResponse.ts (hunk 6 above)
Comment (Pnom) [reply src/obis-observer/commands/uplink/GetObisIdListResponse.ts]: fixed

File: src/obis-observer/commands/downlink/UpdateImageVerify.ts [hunk 7]
```diff
@@ -60,7 +60,9 @@ class UpdateImageVerify extends Command {
 
     // returns full message - header with body
     toBytes (): Uint8Array {
-        return Command.toBytes(COMMAND_ID, new Uint8Array([this.parameters.requestId]));
+        return Command.toBytes(COMMAND_ID,
```
Comment (DarkPark) [inline src/obis-observer/commands/downlink/UpdateImageVerify.ts:63]: COMMAND_ID to a new line

File: tests/obis-observer/message.test.ts.bak (hunk 1 above)
Comment (DarkPark) [reply tests/obis-observer/message.test.ts.bak:1]: still can see it

Comment (Pnom) [reply tests/obis-observer/message.test.ts.bak]: finally did it

**Review by DarkPark — APPROVED** [review]

**Pull request merged**